keras/random/seed_generator.py

```diff
@@ -66,13 +73,14 @@ def seed_initializer(*args, **kwargs):
             dtype = kwargs.get("dtype", None)
             return self.backend.convert_to_tensor([seed, 0], dtype=dtype)
 
-        self.state = self.backend.Variable(
-            seed_initializer,
-            shape=(2,),
-            dtype="uint32",
-            trainable=False,
-            name="seed_generator_state",
-        )
+        with self._open_name_scope():
```

Comment:
This isn't needed, instead you can just do `with backend.name_scope(self.name, caller=self):`

Reply:
Fixed